src/engine/ReluConstraint.cpp

```diff
@@ -829,6 +864,21 @@ unsigned ReluConstraint::getAux() const
     return _aux;
 }
 
+double ReluConstraint::computePolarity() const
+{
+    double currentLb = _lowerBounds[_b];
+    double currentUb = _upperBounds[_b];
+    if ( currentLb >= 0 || currentUb <= 0 ) return 1;
+    double width = currentUb - currentLb;
+    double sum = currentUb + currentLb;
+    return sum / width;
```

Comment:
do we really need to divide the sum by width? isn't sum enough?

Reply:
We just need the sign for direction heuristic. However, we need this to compute how symmetric the bound is with respect to zero in the relu-splitting branching heuristic for DnC. Concretely, we will pick ReLUs with the polarity that is closest to 0.